plugins/continuous-toolbox/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@blockly/continuous-toolbox",
+  "version": "0.0.0",
+  "description": "A Blockly plugin that adds a continous-scrolling style toolbox and flyout",
+  "scripts": {
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run build",
```

Comment:
Add two more scripts:
`"audit:fix": "blockly-scripts auditFix",`
`"predeploy": "npm run build && blockly-scripts predeploy",`

We've added those since you built this plugin.

Reply:
We should add these to the create-package script too.